R/extract.R

```diff
@@ -4434,19 +4434,19 @@ extract.vglm <- function(model, include.loglik = TRUE, include.df = TRUE,
 	return(tr)
 }
 
-setMethod("extract", signature = className("vglm", "VGAM"), 
+setMethod("extract", signature = className("vglm", "VGAM"),
     definition = extract.vglm)
 
 
 # extension for weibreg objects (eha package)
-extract.weibreg <- function(model, include.loglik = TRUE, include.lr = TRUE, 
+extract.weibreg <- function(model, include.aic = TRUE, include.loglik = TRUE, include.lr = TRUE,
```

Comment:
make sure this is within a width of 80 characters -- here and in all other places you edited